lib/types/types.go

```diff
@@ -230,7 +230,7 @@ func (d NullDuration) ValueOrZero() Duration {
 
 // TimeDuration returns a NullDuration's value as a stdlib Duration.
 func (d NullDuration) TimeDuration() time.Duration {
-	return time.Duration(d.Duration)
+	return d.TimeDuration()
```

Comment:
This should be reverted as this is just calling the same function, effectively making an endless recursion

Reply:
Thank you so much! This is reverted in 047c231